<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BranchAlarmSet</class>
 <widget class="QWidget" name="BranchAlarmSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>390</y>
      <width>96</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>支路告警设置</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>666</width>
      <height>285</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>支路</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>告警</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>支路</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>告警</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>支路</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>告警</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>支路</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>告警</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>390</y>
      <width>96</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
